### PR TITLE
Implement mobile navigation bar

### PR DESCRIPTION
**Pull request by reschan**
Mobile view only.
image
image
image

Navbar also closes and open depend on scroll speed.

### CODE REVIEW

File: js/utils.js [hunk 1]
```diff
@@ -952,3 +952,39 @@ function make_SCC_graph(root_node, nodes) {
...
     return sccs;
 }
+
+
+// Toggles display of a certain element, given the ID.
+function toggle_tab(tab) {
+    if (document.querySelector("#"+tab).style.display == "none") {
+        document.querySelector("#"+tab).style.display = "";
+    } else {
+        document.querySelector("#"+tab).style.display = "none";
```
Comment (hppeng-wynn) [inline js/utils.js:962]: Performance: grab the thing by queryselector once and then use it

Also is this equivalent to `document.getElementById`? better to be explicit

**Review by hppeng-wynn — CHANGES_REQUESTED** [review]
Is `icons.js` still doing anything? Can we use that to reduce code duplication?

File: js/utils.js (hunk 1 above)
Comment (reschan) [reply js/utils.js]: fixed

**Review by hppeng-wynn — APPROVED** [review]
lgtm

**Pull request merged**